sdk/monitor/Azure.Monitor.Query/src/LogsQueryClientBuilderExtensions.cs

```diff
@@ -12,7 +12,15 @@ public static class LogsQueryClientBuilderExtensions
     {
         /// <summary> Registers a <see cref="LogsQueryClient"/> instance. </summary>
         /// <param name="builder"> The builder to register with. </param>
-        /// <param name="endpoint"> The resource manager service endpoint to use. For example <c>https://management.azure.com/</c> for public cloud. </param>
+        public static IAzureClientBuilder<LogsQueryClient, LogsQueryClientOptions> AddLogsQueryClient<TBuilder>(this TBuilder builder)
+        where TBuilder : IAzureClientFactoryBuilderWithCredential
+        {
+            return builder.RegisterClientFactory<LogsQueryClient, LogsQueryClientOptions>((options, cred) => new LogsQueryClient(cred, options));
+        }
+
+        /// <summary> Registers a <see cref="LogsQueryClient"/> instance. </summary>
+        /// <param name="builder"> The builder to register with. </param>
+        /// <param name="endpoint"> The service endpoint to use. Default is 'https://api.loganalytics.io'. </param>
```

Comment:
We should mention the default in the new method that is being added. For this one there is no need to mention the default, as the user is required to pass the endpoint.